<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kompis</class>
 <widget class="QMainWindow" name="Kompis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KompisUSB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Velg modus:</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SA OPPSETT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>RTG OPPSETT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SERVICE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>NULLSTILL</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Diverse:</string>
    </property>
    <widget class="QCheckBox" name="box_skype">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Skype</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_steam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Steam</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_7zip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>7-Zip</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_imgburn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ImgBurn</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Rens:</string>
    </property>
    <widget class="QCheckBox" name="box_malw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>MalwareBytes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_crapfjerner">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CrapFjerner</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_ccleaner">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CCleaner</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_onyx">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>OnyX</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/files/img/elkjop.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Oppdatering:</string>
    </property>
    <widget class="QCheckBox" name="box_win10update">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Win10 Update</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_win8update">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Win8.1 Update</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_win8update_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>WSUS Offline</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Verkt√∏y:</string>
    </property>
    <widget class="QCheckBox" name="box_classicstart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Classic Start</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_mailpw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>MailPW</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_browserpw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>BrowserPW</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_produkey">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ProduKey</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_ddu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DDU</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_produkey_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reveal</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Avinstaller:</string>
    </property>
    <widget class="QCheckBox" name="box_mcafee">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>McAfee Livesafe</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_norton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Norton Security</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_fsecure">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>F-Secure</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_fjerncloud">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Elkj√∏p Cloud</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_fjernmalw">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>MalwareBytes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_fjernremote">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RemoteFix</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Internett:</string>
    </property>
    <widget class="QCheckBox" name="box_mozilla">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mozilla Firefox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_chrome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Google Chrome</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_elkjopcloud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Elkj√∏p Cloud</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_remotefix">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RemoteFix</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Media:</string>
    </property>
    <widget class="QCheckBox" name="box_vlc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>VLC</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_itunes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>iTunes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_spotify">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Spotify</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_codecs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Codecs</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Kontor:</string>
    </property>
    <widget class="QCheckBox" name="box_libreoffice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LibreOffice</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_office">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Microsoft Office</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_winlive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Windows Live Mail</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_thunderbird">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Thunderbird</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_adobereader">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Adobe Reader</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_foxit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Foxit Reader</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Runtime:</string>
    </property>
    <widget class="QCheckBox" name="box_java">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Java</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_silverlight">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Silverlight</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_dotnet">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>.NET</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_adobeair">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Adobe Air</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="logBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>701</width>
      <height>221</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/files/img/kh.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Avslutning:</string>
    </property>
    <widget class="QCheckBox" name="box_skype_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Snarveier</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_steam_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Kunde info</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_skype_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sjekk nettleser</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="box_skype_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Restart</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startKompis">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>440</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>START KOMPIS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMeny">
    <property name="title">
     <string>Meny</string>
    </property>
    <addaction name="menu_innstillinger"/>
    <addaction name="menu_avslutt"/>
   </widget>
   <widget class="QMenu" name="menuOm">
    <property name="title">
     <string>Om</string>
    </property>
    <addaction name="menu_oppdatering"/>
    <addaction name="menu_omkompis"/>
   </widget>
   <addaction name="menuMeny"/>
   <addaction name="menuOm"/>
  </widget>
  <action name="menu_innstillinger">
   <property name="text">
    <string>Innstillinger</string>
   </property>
  </action>
  <action name="menu_avslutt">
   <property name="text">
    <string>Avslutt</string>
   </property>
  </action>
  <action name="menu_omkompis">
   <property name="text">
    <string>Om Kompis</string>
   </property>
  </action>
  <action name="menu_oppdatering">
   <property name="text">
    <string>Se etter oppdatering</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
